### ISSUE TITLE
Feature Request: Thank you for making SSR.  I use it all the time.  Please consider a timer function so that SSR could could be set to record or shutoff at specific times. 

### ISSUE THREAD

**Issue by griffmky** [MaartenBaert/ssr#886]

**Comment by mark2185** [comment]
That already exists. Be sure to use a recent enough version.

![image](https://user-images.githubusercontent.com/8365241/119137345-038aca00-ba41-11eb-83f0-73f37794ce69.png)

(Edit schedule)

**Comment by mark2185** [comment]
@griffmky is this the feature you requested?

**Comment by fxmarty** [comment]
@mark2185 @MaartenBaert To the best of my knowledge (0.4.2), there is the option to schedule a "start" or "pause", but not a "save recording". Am I correct? It could be handy to have.

**Comment by Charlie-Ramirez-Animation-Studios-de-MX** [comment]
> _No description provided._
@griffmky

Yep, as the other colleagues said, the function of programming a specific start and cut-off time has been available for a long time, (although it is more a pause but separable by deactivating the option to record in a single file; i guess). **However, it seems to me that it is a "problem" of the versions that can be obtained in the app stores of some distros** (one of them being Ubuntu and if I remember correctly some of Debian)  _although it is strange because it seems that some distros deactivate this function in their APT compilations despite being apparently the same current one or only 1 version back_  [i'm Debian 11 but I use manual compilation], this because they are not updated so regularly (and i can't use ubuntu ppa on debian) that it is preferable to compile the latest version from Github.  

to compile manually the instructions are in the repository readme
Here I must admit that @MaartenBaert (i guess) has done a good job of making manual compiling easy and semi-auto for beginners _(Compared to other programs that don't even have the Make file created)_

![Screenshot_20211122_160029](https://user-images.githubusercontent.com/64709446/142942264-8daa3866-97cb-47b0-bafd-57660e2b00d3.png)
